gradle-plugin/src/main/kotlin/com/google/devtools/ksp/gradle/KspAATask.kt

```diff
@@ -59,6 +63,7 @@ abstract class KspAATask @Inject constructor(
         }
```

Comment:
Should be able to replace this with noIsolation now, for better performance.

Reply:
Actually changes to `noIsolation` causes runtime errors inside KSPAATask itself due to classes like `KSPLogger` not found.

Reply:
That is actually an indication of missing libraries in the classpath: `symbol-processing-api`